main.go

```diff
@@ -41,9 +43,17 @@ func run() error {
 	flag.Parse()
 	now := time.Now().UTC()
 
+	repoOwner := os.Getenv("VULNLIST_REPOSITORY_OWNER")
```

Comment:
It seems that LookupEnv is better. In addition, this process should be functionalized like utils.LookupEnv(key, defaultValue string).

Reply:
fixed

Reply:
Would you create new function such as `utils.LookupEnv(key, defaultValue string)`?

Reply:
func LookupEnv(key, defaultValue string){
    if val, ok := os.LookupEnv(key); ok {
		return val
    }
    return defaultValue
}